dss_client/test/dotnet_test_app/test_dss_cwrapper.c

```diff
@@ -0,0 +1,81 @@
+#include <stdio.h>
```

Comment:
Why do we need this file in the first place. You can get the function names in DLLImport and call those functions directly from dotnet rather than one more layer